packages/evolution-backend/src/services/routing/index.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+
+// This file contains route calculation utilities that can be used by the
+// survey.
+import moment from 'moment';
+import { RoutingOrTransitMode } from 'chaire-lib-common/lib/config/routingModes';
+import { getPointCoordinates } from 'chaire-lib-common/lib/services/geodata/GeoJSONUtils';
+import { RouteCalculationParameter, RoutingTimeDistanceResultByMode } from './types';
+import projectConfig from '../../config/projectConfig';
+import routeWithTransitionApi from './RouteCalculationFromTransition';
+
+/**
+ * Calculate the routes for a list of modes
+ * @param modes The list of modes to calculate the routes for
+ * @param parameters The parameters for the route calculation
+ * @param withGeojson Whether to include the GeoJSON in the response
+ */
+export const calculateRoutesByMode = async function (
```

Comment:
The name is wrong right?  We do not calculate route, we would calculate route statistics ?